pennylane_lightning/core/bindings/Bindings.hpp

```diff
@@ -493,7 +493,7 @@ void registerBackendAgnosticMeasurements(PyClass &pyclass) {
                      strides /* strides for each axis     */
                      ));
              })
-        .def("set_seed", [](Measurements<StateVectorT> &M, std::size_t seed) {
+        .def("set_random_seed", [](Measurements<StateVectorT> &M, std::size_t seed) {
```

Comment:
Do we know that this function name change doesn't affect any internal/external people?